app/assets/javascripts/application.js

```diff
@@ -392,24 +392,40 @@ function foreman_url(path) {
 }
 
 $.fn.indicator_show = function(){
- $(this).parents('.form-group').find('img').show();
+ $(this).parents('.form-group').find('.spinner').show();
 }
 
 $.fn.indicator_hide = function(){
- $(this).parents('.form-group').find('img').hide();
+ $(this).parents('.form-group').find('.spinner').hide();
 }
 
 function spinner_placeholder(text){
   if (text == undefined) text = "";
-  return "<div class='spinner-placeholder'>" + text + "</div>"
+  return "<div class='spinner-placeholder'><p class='spinner-label'>" + text + "</p><div id='Loading' class='spinner spinner-md spinner-inline'> </div></div>";
 }
 
 function notify(item, type) {
-  var options = { type: type, sticky: (type != 'success') };
-  $.jnotify($(item).text(), options);
+  var icon = typeToIcon(type);
+  var options = { classMessage: "alert alert-" + type,
```

Comment:
note: jnotify does not support multiple classes passed via classMessage, maybe we should simply add the class via jquery instead?